dpnp/dpnp_algo/dpnp_algo.pyx

```diff
@@ -65,7 +65,6 @@ include "dpnp_algo_counting.pxi"
 include "dpnp_algo_indexing.pxi"
 include "dpnp_algo_linearalgebra.pxi"
 include "dpnp_algo_logic.pxi"
-include "dpnp_algo_manipulation.pxi"
```

Comment:
I had to remove `${CMAKE_CURRENT_SOURCE_DIR}/dpnp_algo_manipulation.pxi` from `/dpnp/dpnp_algo/CMakeLists.txt` to be able to compile the code, did not you?

I also had to remove `cpdef dpnp_descriptor dpnp_repeat(dpnp_descriptor array1, repeats, axes=*)` from `dpnp/dpnp_algo/dpnp_algo.pxd`.